tox.ini

```diff
@@ -48,6 +48,7 @@ passenv =
     dev: NGINX_SERVER
     dev: CLIENT_EMBED_URL
     dev: SIGNED_URLS_REQUIRED
+    dev: JSTOR_PDF_URL
```

Comment:
It would be a good idea to re-sort these variables, unless there is a reason they need to be in this particular order?